sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/firebase.yaml

```diff
@@ -16,11 +16,19 @@ provider:
     - id: legacy-fcm-api-key
       rspecKey: S6710
       metadata:
-        name: Legacy Firebase Cloud Messaging API Key
+        name: Legacy Firebase Cloud Messaging API Keys
       detection:
         matching:
-          pattern: "(?i)\\b([a-z0-9\\-_]{11}:apa91b[a-z0-9\\-_+]{134})\\b"
+          pattern: "(?i)\\b([\\w\\-]{11}:apa91b[\\w\\-+]{134})\\b"
       examples:
+        - text: |
+            # Noncompliant code example
```

Comment:
The noncompliant code example should raise an issue. Maybe the example needs to be adjusted?

Reply:
Done, and adjusted here: https://github.com/SonarSource/rspec/pull/3128